src/Tribe/Global_Stock.php

```diff
@@ -62,10 +99,15 @@ public function __construct( $post_id ) {
 	 * As a convenience, false can be passed to this method to disable rather
 	 * than enable global stock.
 	 *
-	 * @param bool $yes
+	 * @since  4.1
+	 * @since  TBD Added a return so we can check if it was enabled correctly
+	 *
+	 * @param  bool $yes
+	 *
+	 * @return bool
```

Comment:
Should be `int|bool`? (If the `self::GLOBAL_STOCK_ENABLED` key didn't previously exist, `update_post_meta()` will return the ID)